Assets/_BForBoss/_Weapons/Scripts/Weapons/WeaponBehaviour.cs

```diff
@@ -81,12 +81,18 @@ private void HandleOnStopReloading()
         {
             _weaponAnimationProvider.ReloadingWeapon(false);
         }
+        
+        protected virtual void HandleOnStartReloading()
```

Comment:
protected should be above private

Reply:
realized this whole class has a mishmash of protected and private, will leave it up to you if you want to change things around